devicetypes/sky-nie/minoston-wallmote.src/minoston-wallmote.groovy

```diff
@@ -0,0 +1,318 @@
+/**
```

Comment:
Why PR's title contains `Aeotec` brand?

Reply:
I am also puzzled by this: this DTH is designed with reference to Aeotec Wallmote. My original plan was to directly add it to the original file, but later found that there are too many places to be changed (currently not fully implemented).

Reply:
@sky-nie 
_"currently not fully implemented"_
What else is gonna be implemented here ?